docs/source/_templates/layout.html

```diff
@@ -81,7 +81,8 @@
       <div class="nav__item nav__item--brand desktop_only">
         <a
           class="button-primary"
-          onclick="open_modal('early-access', 'close-early-access')"
+          onclick="https://voxel51.com/#teams-form"
```

Comment:
@benjaminpkane After a clean docs build, clicking on this button in the header doesn't do anything for me

Reply:
Fixed, my mistake